lib/littlebox.css

```diff
@@ -3261,26 +3261,26 @@
 .lb-flag-france {
```

Comment:
Probably still good to get them out of there though ha. They did not look good. Thanks for checking it out!